example/example.py

```diff
@@ -48,14 +49,15 @@ def evaluate(kwargs: EvaluateInput) -> pandas.DataFrame:
 
 @task
 def translate(args: Tuple[FilePath, DirectoryPath]) -> DirectoryPath:
-    model, sentences = args
+    model_dir, sentences = args
     task_ctx = task.context()
-    with open(model) as f, open(sentences / "file.txt") as g:
+    with open(model_dir / "translate_model.txt") as f, open(
+        sentences / "file.txt"
+    ) as g:
         print(
             "translate({}, {}, config={})".format(
-                f.read().strip(),
-                g.read().strip(),
-                task_ctx.config)
+                f.read().strip(), g.read().strip(), task_ctx.config
```

Comment:
f.read().strip(),
g.read().strip(),
task_ctx.config

new lines will be similar to earlier standard